### PR TITLE
fix: added release workflow

### PR DESCRIPTION
**Pull request by abhinav-from-contentstack**
Current version: 0.0.3
Updated version: 0.0.4

This PR introduces some vulnerability fixes along with addition of a release workflow and an updated LICENSE file. Hence, a patch version update is needed. 

Therefore, I've updated the package version to 0.0.4

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+          prerelease: false # Default value, but nice to set explicitly
+      - name: Upload Release Asset
+        if: steps.update_tag.outputs.tagname
+        id: upload-release-asset
+        uses: actions/upload-release-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          upload_url: ${{ steps.create_release.outputs.upload_url }} # This pulls from the CREATE RELEASE step above, referencing its ID to get its outputs object, which include a `upload_url`. See this blog post for more info: https://jasonet.co/posts/new-features-of-github-actions/#passing-data-to-future-steps
+          asset_path: ./datasync-filesystem-sdk-${{ steps.package-version.outputs.current-version }}.tgz
```
Comment (netrajpatel) [inline .github/workflows/release.yml:48]: @abhinav-from-contentstack Why is this workflow using datasync-filesystem-sdk?

**Review by netrajpatel — CHANGES_REQUESTED** [review]

File: .github/workflows/release.yml (hunk 1 above)
Comment (abhinav-from-contentstack) [reply .github/workflows/release.yml]: I must've missed it @netrajpatel. I've added a correction. Please check.

**Review by netrajpatel — CHANGES_REQUESTED** [review]
@abhinav-from-contentstack 
Please add a note for plugin version update

**Comment by abhinav-from-contentstack** [comment]
done @netrajpatel

**Review by netrajpatel — DISMISSED** [review]

**Comment by netrajpatel** [comment]
@vkalta There are SCA scans which are failing in this PR, can you please fix those?

**Review by aravindbuilt — APPROVED** [review]

**Review by netrajpatel — APPROVED** [review]

**Pull request merged**